### DISCUSSION TITLE
can someone help me make a github portfolio

### DISCUSSION THREAD

**Discussion by SakuraKitsunee** [Prodigy-Hacking/ProdigyMathGameHacking#844] [(A) Anything]
can someone help me make a github pages portfolio? if you could that would be great

thx, 

UN3N0WNGH0ST

**Comment by irhrhd** [comment]
Dont you know how to code? After all you got mod because you can do stuff
right?

**Reply by SakuraKitsunee** [reply]
i do but not html

**Comment by SakuraKitsunee** [comment]
i can code java and other stuff like that

**Comment by irhrhd** [comment]
Plus dont make issues about your personal wants please close this andd do
it in the discussions

**Reply by LeoBadeaux** [reply]
> Plus dont make issues about your personal wants please close this andd do it in the discussions

irhrhd, this is the anything category. let us do our jobs.

**Comment by SakuraKitsunee** [comment]
this is a discussion tho

**Comment by irhrhd** [comment]
hmm guess it is

**Comment by irhrhd** [comment]
i was on my email and all i saw was the numbers #844

**Comment by SakuraKitsunee** [comment]
huh?

**Reply by irhrhd** [reply]
thought it was an issue my bad

**Comment by irhrhd** [comment]
but why cant you just learn to code html its really easy i have other things to do than to make stuff for other people i have a job called coding a game and website jumpbound (digitalteal)

**Comment by SakuraKitsunee** [comment]
oh it is do you have any website or vids that can teach it

**Comment by irhrhd** [comment]
what ever ill do it what do you want done

**Comment by SakuraKitsunee** [comment]
oh ok sure

**Comment by SakuraKitsunee** [comment]
ill invite you as a collaborator!

**Comment by irhrhd** [comment]
Lets continue this on dc RASBeRrY#0067 is my tag

**Comment by SakuraKitsunee** [comment]
i just did it

**Comment by SakuraKitsunee** [comment]
i cant go on discord rn

**Comment by ghost** [comment]
I mean, sure, but you gotta pay me :D

Edit: oh nvm you have RAS coding sad

**Comment by ghost** [comment]
щ(゜ロ゜щ)

**Comment by SakuraKitsunee** [comment]
hey my friend and myself made one so does anyone want to check it out?
